content/en/docs/reference/networking/virtual-ips.md

```diff
@@ -126,6 +126,28 @@ iptables:
 ...
 ```
 
+##### Experimental performance improvements {#minimize-iptables-restore}
+
+{{< feature-state for_k8s_version="v1.27" state="beta" >}}
+
+In Kubernetes 1.27, new performance improvements to the iptables proxy
+mode are enabled by default, which should make it much less necessary to
+override the default syncing timeouts.
+
+If you were previously overriding `minSyncPeriod`, you should try
+removing that override and letting kube-proxy use the default value
+(`1s`) or at least a smaller value than you were using before.
+
+If you notice kube-proxy's
+`sync_proxy_rules_iptables_restore_failures_total` or
+`sync_proxy_rules_iptables_partial_restore_failures_total` metrics
+increasing after upgrading to 1.27, that likely indicates you are
+encountering bugs in the new code, and you should file a bug report.
+(You can disable the new optimizations by disabling the
+`MinimizeIPTablesRestore` [feature
+gate](/docs/reference/command-line-tools-reference/feature-gates/) for
+kube-proxy with `--feature-gates=MinimizeIPTablesRestore=false,…`.)
```

Comment:
We avoid documenting transitions between versions within the reference docs (exception: https://kubernetes.io/docs/reference/using-api/deprecation-guide/ and friends). As docs and features move towards GA, this approach becomes more important.

As a result, this information probably belongs in the post-release blog article _and_ in the v1.27 release notes.
The PR that adds the blog article can also update this page to link there, and we can merge that on release day provided that folks plan ahead and get everything ready for review in plenty of time.

Reply:
ok, it's already in the release note for https://github.com/kubernetes/kubernetes/pull/115138

Reply:
The next step is to cut out or reword this section - otherwise, you'll need to commit SIG Network to maintaining this and updating it for the v1.28 release, when this wording will be stale.